.github/workflows/run-update-stage.yml

```diff
@@ -0,0 +1,62 @@
+      - uses: actions/checkout@v2
+      - name: Install Python & dependencies.
+        run: |
+          sudo apt-get update
+          sudo apt-get install python3 python3-dev python3-pip -y
+          sudo pip3 install -r requirements.txt
+      - name: Run config file generator.
+        run: |
+          PYTHONPATH=`pwd` python3 miner_config/generate_config.py
+          # Output latest version info
+          HEIGHT=$(curl -s -H 'Cache-Control: no-cache' https://storage.googleapis.com/helium-snapshots-stage.nebra.com/latest.json | jq -r '.height')
+          HASH=$(curl -s -H 'Cache-Control: no-cache' https://storage.googleapis.com/helium-snapshots-stage.nebra.com/latest.json | jq -r '.hash')
+          echo "Block height: $HEIGHT"
+          echo "Block hash: $HASH"
+          exit 0
+      - name: Setup gsutil.
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          service_account_key: ${{ secrets.GOOGLE_SA }}
+          project_id: ${{ secrets.GOOGLE_PROJECT }}
+          export_default_credentials: true
+      - name: Copy config file to bucket.
+        run: |
+          gsutil cp docker.config gs://helium-assets-stage.nebra.com/docker.config
+      - name: Set ACL
+        run: |
+          gsutil acl set acl.txt gs://helium-assets-stage.nebra.com/docker.config
+      - name: Change Cache-Control metadata of assets
+        run: |
+          gsutil setmeta -h "Cache-Control:max-age=60" gs://helium-assets-stage.nebra.com/docker.config
+          gsutil setmeta -h "Cache-Control:max-age=60" gs://helium-snapshots-stage.nebra.com/latest.json
+          gsutil setmeta -h "Cache-Control:max-age=60" gs://helium-snapshots-stage.nebra.com/latest-snap.json
+      - name: Purge Cloudflare Cache for URL
+        run: |
+          sleep 2m
+          curl -X POST "https://api.cloudflare.com/client/v4/zones/${{ secrets.CLOUDFLARE_ZONE_ID }}/purge_cache" \
+          -H "Authorization: Bearer ${{ secrets.CLOUDFLARE_CACHE_PURGE_KEY }}" \
+          -H "Content-Type: application/json" \
+          --data '{"files":["https://helium-assets.nebra-stage.com/docker.config",{"url":"https://helium-assets-stage.nebra.com/docker.config"}]}'
+          curl -X POST "https://api.cloudflare.com/client/v4/zones/${{ secrets.CLOUDFLARE_ZONE_ID }}/purge_cache" \
+          -H "Authorization: Bearer ${{ secrets.CLOUDFLARE_CACHE_PURGE_KEY }}" \
+          -H "Content-Type: application/json" \
+          --data '{"files":["https://helium-snapshots-stage.nebra.com/latest.json",{"url":"https://helium-snapshots-stage.nebra.com/latest.json"}]}'
+          curl -X POST "https://api.cloudflare.com/client/v4/zones/${{ secrets.CLOUDFLARE_ZONE_ID }}/purge_cache" \
+          -H "Authorization: Bearer ${{ secrets.CLOUDFLARE_CACHE_PURGE_KEY }}" \
+          -H "Content-Type: application/json" \
+          --data '{"files":["https://helium-snapshots-stage.nebra.com/latest-snap.json",{"url":"https://helium-snapshots-stage.nebra.com/latest.json"}]}'
```

Comment:
Should it be `latest-snap.json` at the very end?

Reply:
Yes! Good catch.